doc/republish_all_content.md

```diff
@@ -0,0 +1,14 @@
+# Republish all content from the Publishing API
```

Comment:
It would be great if this could be described without using "republish", to avoid confusion with the existing meanings of "republish" (for example, [1][2]). How about "Re-send", "Re-queue" or "Re-present"?

1: https://github.com/alphagov/publishing-api/blob/master/app/commands/v2/republish.rb
2: https://github.com/alphagov/whitehall/blob/master/lib/tasks/publishing_api.rake#L109-L113

Reply:
Updated - thanks for the suggestion! 1bddfdafc8a44ce5a67e5eb0f832944c72cac10f